orchestration/tests/test_cleanup_completed_agents.py

```
@@ -0,0 +1,768 @@
+        """Test completion marker in log returns completed=True."""
+        # Arrange
+        mock_exists.return_value = True
+        mock_run.return_value = Mock(stdout="Some log content\nClaude exit code: 0\nMore content")
+
+        # Act
+        result = check_agent_completion("test-agent")
+
+        # Assert
+        assert result["completed"] is True
+        assert "found_marker" in result["reason"]
+        assert result["log_path"] == "/tmp/orchestration_logs/test-agent.log"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/orchestration_logs/test-agent.log"

(S108)